### ISSUE TITLE
no-unused-variable is deprecated

### ISSUE THREAD

**Issue by LinusU** [blakeembrey/tslint-config-standard#51]
> no-unused-variable is deprecated. Since TypeScript 2.9. Please use the built-in compiler checks instead.

Should we just disable the rule?

Personally, I'm not a fan of moving lint checks into the compiler, I think... Oh well 🤷‍♂️

**Comment by blakeembrey** [comment]
Me either. I think this was already fixed though, just needs to be released. Will follow up over the weekend.

**Comment by LinusU** [comment]
Yeah, looks like it was fixed here: 1a99e55bbe24553331fa78b8c088f9cb0b06f113

**Comment by blakeembrey** [comment]
Is this somewhat urgent for you? As far as I understand it’s a deprecation, is it breaking something?

**Comment by jrobeson** [comment]
If there were no other changes coming soon then I would want the release. In this case though, might as well wait for a typescript 3.x compatible tslint.

**Comment by Boycce** [comment]
Any chance we publish a new release, linting deprecation notices are a pain in Sublime.

**Comment by jrobeson** [comment]
It'd be nice to see  https://github.com/blakeembrey/tslint-config-standard/pull/53 added before a new release is cut.

**Comment by adrianhopebailie** [comment]
Minor changes are what npm patch releases are for.
A quick patch (v7.1.1) that includes 1a99e55 would be great

**Comment by blakeembrey** [comment]
This would be a major release, not a patch. It’s removing a feature.

**Comment by Pilatch** [comment]
I, too, am distracted by my linter yelling at me each time I open my text editor and point it at a TS file.

It burns my biscuits, but does not fry my loins. Therefore I will wait for 8.
